dotCMS/src/main/java/com/dotcms/experiments/business/ExperimentsAPIImpl.java

```diff
@@ -662,6 +674,29 @@ public boolean isAnyExperimentRunning() throws DotDataException {
                 !APILocator.getExperimentsAPI().getRunningExperiments().isEmpty();
     }
 
+    @Override
+    public void endFinalizedExperiments(final User user) throws DotDataException {
```

Comment:
I think this method should go into the Job

Reply:
why?